### PR TITLE
CSSRE-1979: added github-action for upstream-changes 

### PR DESCRIPTION
**Pull request by vinaybommana**
Hi @deepak1725
I've added the following github action to fetch the upstream changes every monday on schedule, create a new branch - upstream-changes- and raises a pull request.
based on our previous discussion, please suggest default reviewers i should add automatically for the raised PR

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        upstream_sync_repo: https://gitlab.com/glitchtip/glitchtip-frontend.git
+
+        # Set test_mode true to run tests instead of the true action!!
+        test_mode: false
+
+    - name: Create Pull Request if new commits are present
+      if: steps.sync.outputs.has_new_commits == 'true'
+      uses: actions/github-script@v6
+      with:
+        script: |
```
Comment (deepak1725) [inline .github/workflows/main.yml:42]: Can we create a standalone script file and call it here?

**Review by deepak1725 — CHANGES_REQUESTED** [review]

**Comment by vinaybommana** [comment]
Hi @deepak1725

- added the simple bash script to create a upstream-changes- branch
- the github action creates a PR
- added CODEOWNERS file as the cs-sre team.
- added another github action which added given reviewers to the newly opened PR, for now I've added the suggested reviewers

please give it a look once

File: .github/workflows/main.yml (hunk 1 above)
Comment (vinaybommana) [reply .github/workflows/main.yml:45]: this is done

**Comment by deepak1725** [comment]
Looks good. Make sure to fix the Validation Errors visible in changelog, 
Say EOL errors and likewise.

**Review by deepak1725 — APPROVED** [review]
LGTM

**Pull request merged**